src/common/transformations/tests/common_optimizations/flush_fp32_subnormals_to_zero_test.cpp

```diff
@@ -0,0 +1,382 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pass/manager.hpp>
+#include <transformations/flush_fp32_subnormals_to_zero.hpp>
+#include <transformations/init_node_info.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "ngraph/pass/visualize_tree.hpp"
+
+using namespace testing;
+using namespace ngraph;
+using namespace std;
+
+namespace {
+union FloatIntUnion {
+    uint32_t u;
+    float f;
+};
+FloatIntUnion maximum_subnorm_val = {0x007fffff};  // = 2^−126 * (1 - 2^−23) ~= 1.1754942107e-38f
+FloatIntUnion minimum_subnorm_val = {0x00000001};  // = 2^−149 ~= 1.4012984643e-45f
+FloatIntUnion minimum_norm_val = {0x00800000};     // = 2^−126 ~= 1.1754943508-38f
+}  // namespace
```

Comment:
please use limits, all of these values are defined: https://en.cppreference.com/w/cpp/types/numeric_limits/denorm_min
please replace all hard-coded numbers in PR.

Reply:
Unfortunately there are all numbers, except `subnormal_max`. Can we leave this test as is?